tests/helper/token-stream-transformer.js

```diff
@@ -20,13 +20,53 @@ module.exports = {
 				.filter(function (value) {
 					return !(Array.isArray(value) && !value.length) && !(typeof value === "string" && !value.trim().length);
 				}
-			);
+				);
```

Comment:
Nit: this should be on the same line as the closing `}`.

Reply:
Done.